docker/scripts/scripts/httpsender/Alert_on_Unexpected_Content_Types.js

```diff
@@ -9,25 +9,15 @@ var pluginid = 100001	// https://github.com/zaproxy/zaproxy/blob/main/docs/scann
 var extensionAlert = control.getExtensionLoader().getExtension(org.zaproxy.zap.extension.alert.ExtensionAlert.NAME)
 
 var expectedTypes = [
-		"application/hal+json",
-		"application/health+json",
-		"application/json",
 		"application/octet-stream",
-		"application/problem+json",
-		"application/problem+xml",
-		"application/soap+xml",
-		"application/vnd.api+json",
-		"application/xml",
-		"application/x-ndjson",
-		"application/x-yaml",
-		"application/yaml",
-		"text/x-json",
-		"text/json",
-		"text/yaml",
+		"text/html",
```

Comment:
This and CSS are not expected at least not generally, this is for APIs.

Reply:
Okay, I was wondering what I was missing.